plugins/kubernetes-common/report.api.md

```diff
@@ -432,6 +432,10 @@ export interface ResourceRef {
   namespace: string;
 }
 
+// @public
+export const SERVICEACCOUNT_CA_PATH =
+  '/var/run/secrets/kubernetes.io/serviceaccount/ca.crt';
+
```

Comment:
Is there a need to export this as public API? I'd rather keep it internal

Reply:
Updated, thanks!